README.md

```diff
@@ -45,86 +42,105 @@ Get the following:
-
-```bash
-sudo ./requirementsCheck.sh
-```
+>
+> ```bash
+> sudo ./requirementsCheck.sh
+> ```
 
 ## Installation
 
 1. Create a new directory and navigate to it in a terminal.
 
-```bash
-mkdir smartkickers
-cd smartkickers
-```
+   ```bash
+   mkdir smartkickers
+   cd smartkickers
+   ```
 
 2. Clone the repository.
 
-```bash
-git clone https://github.com/HackYourCareer/SmartKickersAI.git
-```
+   ```bash
+   git clone https://github.com/HackYourCareer/SmartKickersAI.git
+   ```
 
-3. Pull frontend image from GCR
+3. Pull the frontend image from GCR.
 
-```bash
-docker pull ghcr.io/hackyourcareer/smartkickers-frontend:latest
-```
+   ```bash
+   docker pull ghcr.io/hackyourcareer/smartkickers-frontend:latest
+   ```
 
-4. Pull backend image from GCR
+4. Pull the backend image from GCR.
 
-```bash
-docker pull ghcr.io/hackyourcareer/smartkickers-backend:latest
-```
+   ```bash
+   docker pull ghcr.io/hackyourcareer/smartkickers-backend:latest
+   ```
 
 ## Launching
 
-> **NOTE:** Configuration files for the camera view are provided by the developers in the [`SmartKickersAI`](https://github.com/HackYourCareer/SmartKickersAI/tree/main/LocalServer) repository.
+> **NOTE:** The configuration files for the camera view are provided by the developers in the [`SmartKickersAI`](https://github.com/HackYourCareer/SmartKickersAI/tree/main/LocalServer) repository.
 
 ### Launching the server and web application
 
-- Run pulled containers
+Run the pulled containers to launch the server and web application.
 
-  - In a terminal run:
+1. In a terminal run:
 
-  ```bash
-  docker run -p 3000:3000 ghcr.io/hackyourcareer/smartkickers-backend:latest
-  ```
+   ```bash
+   docker run -p 3000:3000 ghcr.io/hackyourcareer/smartkickers-backend:latest
+   ```
 
-  - Open a new terminal and run:
+2. Open a new terminal and run:
 
-  ```bash
-  docker run -p 3007:80 ghcr.io/hackyourcareer/smartkickers-frontend:latest
-  ```
+   ```bash
+   docker run -p 3007:80 ghcr.io/hackyourcareer/smartkickers-frontend:latest
+   ```
 
 ### Launching the ball-tracking software
 
 1. Plug in the camera to the computer.
 
-2. In order to launch the application, navigate to the `SmartKickersAI` repo and double click `start` file.
-
-   Successful connection to the Go server shows the connection details like in the screenshot below:
-
-   ![Node launch image](assets/nodeLaunch.png "Node launch")
-
-3. In your Internet browser, go to [`localhost:3007`](http://localhost:3007/) to see the React application.
-
-This takes you to the SmartKickers React application.
-
-![React application](assets/reactApp.png "React application")
+2. To launch the application, navigate to the `SmartKickersAI` repo and double click the `start` file.
```

Comment:
Double checking: does this `start` file not have an extension?

Reply:
I see.
I can see that now you renamed it from `start.sh` to `start`. Was there a specific reason for it? 
I'd probably propose that you clearly state the extension in the name.